var/spack/repos/builtin/packages/py-libensemble/package.py

```diff
@@ -60,7 +60,8 @@ class PyLibensemble(PythonPackage):
     depends_on("py-mpmath", type=("build", "run"), when="+mpmath")
     depends_on("py-deap", type=("build", "run"), when="+deap")
     depends_on("tasmanian+python", type=("build", "run"), when="+tasmanian")
-    depends_on("py-pyyaml", type=("build", "run"), when="+pyyaml")
+    depends_on("py-pyyaml", type=("build", "run"), when="@:0.10.0")
+    depends_on("py-toml", type=("build", "run"), when="@:0.10.0")
```

Comment:
```suggestion
    depends_on("py-pyyaml", type=("build", "run"), when="@0.10:")
    depends_on("py-pyyaml", type=("build", "run"), when="@:0.9 +extras")
    depends_on("py-tomli", type=("build", "run"), when="@0.10:")
```
Also required in develop